src/tabs/tabset.component.ts

```diff
@@ -8,7 +8,7 @@ import { TabsetConfig } from './tabset.config';
   selector: 'tabset',
   template: `
     <ul class="nav" [ngClass]="classMap" (click)="$event.preventDefault()">
-        <li *ngFor="let tabz of tabs" class="nav-item {{tabz.customClass}}"
+        <li *ngFor="let tabz of tabs" [ngClass]="'nav-item ' + tabz.customClass"
```

Comment:
it can be:
`[ngClass]="['nav-item', tabz.customClass]"`

just to avoid concatenation in the template